demisto_sdk/commands/content_graph/strict_objects/integration.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
+from typing import Optional, Any
+from pydantic import Field
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.common.tools import get_file
+from demisto_sdk.commands.content_graph.strict_objects.base_strict_model import BaseStrictModel, CommonFields, \
+    Argument, Output, Important, ScriptType, BaseIntegrationScript
+
+
+class Configuration(BaseStrictModel):
+    display: Optional[str] = None
+    section: Optional[str] = None
+    advanced: Optional[str] = None
+    default_value: Optional[Any] = Field(None, alias="defaultvalue")
+    name: str
+    type: int
+    required: Optional[bool] = None
+    hidden: Optional[Any] = None
+    hidden_x2: bool = None
+    options: Optional[list[str]] = None
+    additional_info: Optional[str] = Field(None, alias="additionalinfo")
+    display_password: Optional[str] = Field(None, alias="displaypassword")
+    hidden_username: Optional[bool] = Field(None, alias="hiddenusername")
+    hidden_password: Optional[bool] = Field(None, alias="hiddenpassword")
+    from_license: Optional[str] = Field(None, alias="fromlicense")
+    default_value_xsoar: Optional[Any] = Field(None, alias="defaultvalue:xsoar")
+    default_value_marketplace_v2: Optional[Any] = Field(None, alias="defaultvalue:marketplacev2")
+    default_value_xpanse: Optional[Any] = Field(None, alias="defaultvalue:xpanse")
+    default_value_xsoar_saas: Optional[Any] = Field(None, alias="defaultvalue:xsoar_saas")
+    default_value_xsoar_on_prem: Optional[Any] = Field(None, alias="defaultvalue:xsoar_on_prem")
+    name_xsoar: Optional[str] = Field(None, alias="name:xsoar")
+    name_marketplacev2: Optional[str] = Field(None, alias="name:marketplacev2")
+    name_xpanse: Optional[str] = Field(None, alias="name:xpanse")
+    name_xsoar_saas: Optional[str] = Field(None, alias="name:xsoar_saas")
+    name_xsoar_on_prem: Optional[str] = Field(None, alias="name:xsoar_on_prem")
+    required_xsoar: Optional[bool] = Field(None, alias="required:xsoar")
+    required_marketplacev2: Optional[bool] = Field(None, alias="required:marketplacev2")
+    required_xpanse: Optional[bool] = Field(None, alias="required:xpanse")
+    required_xsoar_saas: Optional[bool] = Field(None, alias="required:xsoar_saas")
+    required_xsoar_on_prem: Optional[bool] = Field(None, alias="required:xsoar_on_prem")
+
+
+class IntegrationImportant(Important):
```

Comment:
if an `IntegrationImportant` is only inheriting without extending, why not use `Important` as it is?